### PR TITLE
fix(clusterstatus): Update cli output

### PR DESCRIPTION
**Pull request by lperdereau**

### CODE REVIEW

**Comment by lperdereau** [comment]
Current rendered output
```
  Status: healthy
  VMs: 8
  Templates: 6
  Metrics:
    CPU: 0.01/64(0.0%)
    Memory: 8.0 GiB/314.7 GiB(0.0%)
    Disk: 23.4 GiB/62.6 GiB(0.4%)
  Nodes:
    Offline: 0
    Online: 2
    Unknown: 0
  ```

File: src/pvecontrol/__init__.py [hunk 1]
```diff
@@ -7,16 +7,15 @@
...
-from prettytable import PrettyTable
-from collections import OrderedDict
-from humanize import naturalsize
 
 from pvecontrol.node import PVENode
 from pvecontrol.node import NodeStatus
 from pvecontrol.vm import PVEVm
 from pvecontrol.vm import VmStatus
 from pvecontrol.task import PVETask
+from pvecontrol.utils import _print_tableoutput, _filter_keys
```
Comment (paullaffitte) [inline src/pvecontrol/__init__.py:16]: In regard to [PEP 8](https://peps.python.org/pep-0008/#descriptive-naming-styles), underscore prefixed names suggest an internal/private use. You should rename those I think.

> _single_leading_underscore: weak “internal use” indicator. E.g. from M import * does not import objects whose names start with an underscore.

Comment (lcaflc) [reply src/pvecontrol/__init__.py:16]: Yep _ means "local"  which does not exists in python :/

**Comment by lperdereau** [comment]
@paullaffitte @lcaflc do we need json / yaml output ?

**Review by lcaflc — APPROVED** [review]
All seems OK. approved for dev validation

**Pull request merged**